.travis.yml

```diff
@@ -10,49 +10,60 @@ branches:
 cache:
   yarn: true
 
-stages:
- - basic
- - advanced
- - versions
-
 matrix:
   include:
     - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART=basic
-      stage: basic
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
     - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JOB_PART=lint
-      stage: advanced
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
     - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART=integration
-      stage: advanced
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
     - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JOB_PART=unit
-      stage: advanced
-    - os: osx
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+    - os: linux
       node_js: "10"
-      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART=integration
-      stage: versions
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
     - os: linux
-      node_js: "8"
-      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART=integration
-      stage: versions
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
     - os: linux
-      node_js: "6"
-      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART=integration
-      stage: versions
+      node_js: "10"
+      env: NO_WATCH_TESTS=1 JEST=--maxWorkers=2 JOB_PART="ConfigTestCases.test.js|Stats.test.js -t plugins|output|Stats"
+  allow_failures:
+    - os: osx
   fast_finish: true
 
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Why are you installing yarn? This caused failures in the past

Reply:
hmm.. I'll remove it, but the problem was there before this change...